be/src/storage/lake/tablet_manager.cpp

```diff
@@ -567,6 +578,36 @@ StatusOr<CompactionTaskPtr> TabletManager::compact(int64_t tablet_id, int64_t ve
     return std::make_shared<HorizontalCompactionTask>(txn_id, version, std::move(tablet_ptr), std::move(input_rowsets));
 }
 
+Status TabletManager::abort_txn(int64_t tablet_id, const int64_t* txns, int txns_size) {
+    Status res;
+    // TODO: batch deletion
+    for (int i = 0; i < txns_size; i++) {
+        auto txn_id = txns[i];
+        auto txn_log_or = get_txn_log(tablet_id, txn_id);
+        if (txn_log_or.status().is_not_found()) {
+            continue;
+        } else if (!txn_log_or.ok()) {
+            res.update(txn_log_or.status());
```

Comment:
Status.update() might now sufficient here in case there are more than one failures. Status only reflects the first failure, other failures will not be seen to upper level. Suggest at least include all failed txn_id in status error message.

Reply:
The return value actually will be ignored, I just deleted it:)